### ISSUE TITLE
Update REXML 3.3.3 or greater to resolve denial of service vulnerability 

### ISSUE THREAD

**Issue by ersen-lw** [fastlane/fastlane#22153]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

See https://www.ruby-lang.org/en/news/2024/08/01/dos-rexml-cve-2024-41946/

There is a vulernability in the REXML version that fastlane uses. It should be updated to 3.3.3 or greater to resolve it.

```
Bundler could not find compatible versions for gem "rexml":
  In Gemfile:
    rexml (= 3.3.2)

    fastlane (= 2.221.1) was resolved to 2.221.1, which depends on
      xcodeproj (>= 1.13.0, < 2.0.0) was resolved to 1.24.0, which depends on
        rexml (~> 3.2.4)
```

##### Command executed

N/A

##### Complete output when running fastlane, including the stack trace and command used

N/A

### Environment

N/A

**Comment by ls-philippe-casgrain** [comment]
This should now be REXML 3.3.3 or greater: https://www.ruby-lang.org/en/news/2024/08/01/dos-rexml-cve-2024-41946/

**Comment by TheMetalCode** [comment]
`rexml` is not a _direct_ dependency of fastlane, it's required by [`xcodeproj`](https://github.com/fastlane/fastlane/blob/master/fastlane.gemspec#L109)

https://github.com/CocoaPods/Xcodeproj/pull/948

It looks as though we're just waiting on an `xcodeproj` release.

**Comment by ersen-lw** [comment]
Thanks @TheMetalCode 

It looks like they released 2 hours ago and I was able to update the gem 

```
bundle update xcodeproj

.....
Installing rexml 3.3.4 (was 3.2.8)
Fetching xcodeproj 1.25.0 (was 1.24.0)
```

**Comment by ls-philippe-casgrain** [comment]
It's now 3.3.6... https://www.ruby-lang.org/en/news/2024/08/22/dos-rexml-cve-2024-43398/
